### ISSUE TITLE
Incompatibility with 5.0 due to changes in configurations

### ISSUE THREAD

**Issue by epixa** [elastic/timelion#131]
It looks like timelion needs to be updated to config being moved to the server (I assume).  I got this exception on the latest alpha3 builds:

@bevacqua might be able to help figure this one out.

```
Error: Unexpected `config.get("timelion:showTutorial")` call on unrecognized configuration setting "timelion:showTutorial".
Setting an initial value via `config.set("timelion:showTutorial", value)` before attempting to retrieve
any custom setting value for "timelion:showTutorial" may fix this issue.
getCurrentValue@http://zero:5601/bundles/commons.bundle.js?v=8467:59711:14
[232]/</config.get@http://zero:5601/bundles/commons.bundle.js?v=8467:59591:13
@http://zero:5601/bundles/timelion.bundle.js?v=8467:111663:10
timeout/timeoutId<@http://zero:5601/bundles/commons.bundle.js?v=8467:40195:29
completeOutstandingRequest@http://zero:5601/bundles/commons.bundle.js?v=8467:27847:8
Browser/self.defer/timeoutId<@http://zero:5601/bundles/commons.bundle.js?v=8467:28124:8
```

**Comment by bevacqua** [comment]
I'd say this is a bug in my implementation. Totally my fault. Timelion uses the `config.get(key, defaultValue)` overload, which I didn't think was necessary any longer and removed.

Here you can see the old implementation.

https://github.com/elastic/kibana/blob/4.5/src/ui/public/config/config.js#L79-L90

I'll fix it tomorrow!

**Comment by epixa** [comment]
@bevacqua Will this be a fix in Kibana or in Timelion?

**Comment by bevacqua** [comment]
Kibana

On Wed, May 25, 2016 at 10:16 PM, Court Ewing <notifications@github.com>
wrote:

> @bevacqua <https://github.com/bevacqua> Will this be a fix in Kibana or
> in Timelion?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/timelion/issues/131#issuecomment-221752872>
>



-- 


github <https://github.com/bevacqua> · twitter <https://twitter.com/nzgb> ·
articles <https://ponyfoo.com/> · newsletter <https://ponyfoo.com/weekly> ·
books <https://ponyfoo.com/books> · speaking <https://ponyfoo.com/speaking>
· career <http://careers.stackoverflow.com/bevacqua> · about
<https://ponyfoo.com/about>

**Comment by rashidkpc** [comment]
Fixed in elastic/kibana#7191

**Comment by bevacqua** [comment]
:arrow_heading_up: He means #7291. Merged
